spring-cloud-dataflow-shell-core/src/main/java/org/springframework/cloud/dataflow/shell/command/StreamCommands.java

```diff
@@ -243,6 +245,21 @@ public String updateStream(
 		return String.format("Update request has been sent for the stream '%s'", name);
 	}
 
+	@CliCommand(value = STREAM_SCALE, help = "Scale app instances in a stream")
+	public String scaleStream(
+			@CliOption(key = { "",
+					"name" }, help = "the name of the stream to scale", mandatory = true, optionContext = "existing-stream disable-string-converter") String name,
+			@CliOption(key = {
+					"applicationName" }, help = "the name/label of the application to scale", mandatory = true) String applicationName,
+			@CliOption(key = {
+					"count" }, help = "desired number of application instances", mandatory = true) String count,
+			@CliOption(key = {
+					PROPERTIES_OPTION }, help = "the properties for this scale") String scaleProperties) {
+
+		streamOperations().scaleApplicationInstances(name, applicationName, count, new HashMap<>());
```

Comment:
We can use the properties parsing logic from stream deployment properties option.